CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
 # Releases
 
+## Version 0.14.0
+
+Date: 2022-09-30
+
+This release focuses on three main themes, support for running Panel entirely in the browser using WebAssembly (via Pyodide and PyScript), quality of life improvements to make it easier to build responsive and performant applications and finally improvements to static typing and docstrings to enhance the developer experience.
+
```

Comment:
These changes represent a major step forward in the Panel user and developer experience and the ability to share Panel apps widely without the need for maintaining a central server.